### PR TITLE
Uploading Project Portfolio Site

### PR DESCRIPTION
**Pull request by ogiogiovictor**
First Upload of portfolio Site

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Microverse Porfolio Setup</title>
+    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+  <body>
+
```
Comment (MrRamoun) [inline index.html:11]: - [ ] [REQUIRED] Please, use the `Poppins` font for the heading `<h1>` to match the design as shown in the picture below ⬇️.
- [ ] [REQUIRED] Also, you can adjust the size of the social icons ♻️ to match their counter parts in the original design 🖼️ .

Your website | The figma design
-----|-----
Web capture_21-4-2022_327_rawcdn githack com | image

File: main.css [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social-media span img {
+  width: 21px;
+  height: 20px;
+  padding-top: 10px;
+  padding-right: 10px;
+}
```
Comment (MrRamoun) [inline main.css:87]: - [ ] [REQUIRED] Please, use `flexbox` to layout the social bar icons as mentioned in project requirements 🧾 .

**Review by MrRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi @ogiogiovictor  👋

## Welcome to Microverse! 🌎

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights 🌟

- ✔️ 🎯 Good Use of Git Flow
- ✔️ 🎯 descriptive `README.md` file
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️

- [ ] [REQUIRED] Please, try to stick to the Figma design as possible 🖼️ as it is part of the project requirements 🧾.

_Check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better 👍. So, you can skip them if you don't have the time but remember to come back and implement them because they are going to enhance the quality of your code 💯_

You can also consider:

- [ ] [OPTIONAL] Please, use a more descriptive commit messages 📖 (e.g: instead of `Upload project Site`, you can use `Add header section` - `Add social icons bar` ..etc) ♻️.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **Twitter**: [@ramoun16](https://twitter.com/ramoun16)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1,73 +1,36 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Microverse_Hello_Project
+Microverse Porfolio Project
+This project is a portfolio project design
```
[the remark this answers is not in the document]
Comment (ogiogiovictor) [reply README.md:3]: The Readme File has been updated to have  About, Technologies Used, Getting Started, Authors, Contributions, Support, and Acknowledgment.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+    <section>
+        <div class="main-section-area">
+
+          <div class="main-slide">
+            <h1>I’m Alexandra<br/>
+              Glad to see you!</h1>
+            <p class="slide-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="connect">LET’S CONNECT</p>
```
[the remark this answers is not in the document]
Comment (ogiogiovictor) [reply index.html]: The headline tags have been added to h1, slide-text class and connect class

File: main.css [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+.main-header {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 80px;
+  margin-right: 80px;
+  padding: 20px;
+}
+
+.main-header .Logo {
+  font-family: 'Poppins', sans-serif;
```
[the remark this answers is not in the document]
Comment (ogiogiovictor) [reply main.css]: The display flex did not work using the :root declaration and class but i have replaced the .font with the :root declaration, it worked fine

File: index.html (hunk 1 above)
Comment (ogiogiovictor) [reply index.html:11]: Poppin applied to the h1

File: index.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+      </div>
+    </header>
+
+
+    <section>
+        <div class="main-section-area">
+
+          <div class="main-slide">
+            <h1>I’m Alexandra<br/>
+              Glad to see you!</h1>
```
Comment (ShadyShawkat) [inline index.html:41]: - [ ] [OPTIONAL] While you have the right font family and weight applied to the header title, it doesn't look right.

| Your design | Figma design |
| --- | --- |
| image | image |

- The reason for that is that you're only importing font-weight `200` from google fonts.
https://github.com/ogiogiovictor/microverse_porfolio_project/blob/abd79434833aa2b6b5a15206422cf0da99f9b814/main.css#L1
- You can select multiple font-weight values while selecting the google font to be something like the following: 
```
@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;700&display=swap');
```

**Review by ShadyShawkat — APPROVED** [review]
## Hi @ogiogiovictor, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ogiogiovictor** [comment]
Merging pull request from feature branch

**Pull request merged**